README.md

```diff
@@ -13,11 +13,8 @@ $ flatpak-builder build --install --user ./org.ioquake3.IOQuake3.yaml
 Definitely!
 
 I just need to:
- - … [add an appstream file][1]
- - … [convince upstream to make releases][2]
- - … [ensure I don't keep any crap around][3]
- - … add a flathub.json file (I believe).
```

Comment:
Does ioquake3 work on all arches supported by flathub?

Reply:
The only other active arch at the moment is aarch64 and since my OpenArena Flatpak that also uses ioquake3 works fine there, ioquake3 itself should also work with no issues.